p2p/libp2p/netMessenger.go

```diff
@@ -31,6 +31,9 @@ const DirectSendID = protocol.ID("/directsend/1.0.0")
 const refreshPeersOnTopic = time.Second * 60
 const ttlPeersOnTopic = time.Second * 120
 
+var messageHeader = 200
```

Comment:
is this number always the same ?

Reply:
Not quite. Changed to a higher value (64kB) just to be sure and addes a TODO to remove this message size when https://github.com/libp2p/go-libp2p-pubsub/pull/189 will be part of a new release.